vendors/cypress/psoc6/psoc6make/make/scripts/sign_script.bash

```diff
@@ -0,0 +1,147 @@
+#!/bin/bash
+#
+# This file is used by make to create the build commands to sign an OTA image
+#
+# Modify at your peril !
+#
+# The output files will be in the same directory as the .elf, .hex, etc files
+#
+(set -o igncr) 2>/dev/null && set -o igncr; # this comment is required
+set -e
+
+# Arguments
```

Comment:
Ideally you should be using command-line options, verifying arguments, and providing help if they aren't correct.  Here's a [good stack overflow post](https://stackoverflow.com/questions/192249/how-do-i-parse-command-line-arguments-in-bash) on how to do this in `bash`.

Reply:
Thanks for the feedback.  I will submit an internal ticket to enhance this. It won't be done and verified within the time frame needed here however.